scripts/autoscaling/init_multicloud.sh

```diff
@@ -250,6 +250,12 @@ elif [[ $cloud == *"Microsoft"* ]]; then
         _CLOUD_INSTANCE_IMAGE_ID=""
     fi
     _CLOUD_PROVIDER=AZURE
+
+    CHECK_AZURE_EVENTS_COMMAND="curl -k -H Metadata:true http://169.254.169.254/metadata/scheduledevents?api-version=2017-11-01 2> /dev/null | grep Preempt && kubectl label node $(hostname) preempted=true --kubeconfig='/etc/kubernetes/kubelet.conf'"
```

Comment:
Maybe use something more unique for the label name, e.g. `cloud-pipeline/preempted` or similar? As the `preempted` is becoming a quite common wording.

Reply:
agree